librrd.c

```diff
@@ -148,7 +148,7 @@ json_for_source(RRD_SOURCE * source)
  * Generate JSON for a plugin. This is just a JSON object containing a
  * sub-objecy for every data source. 
  */
-static JSON_Value *
+JSON_Value *
```

Comment:
You are exposing private information to the client of the library. Don't do that.